src/components/layout/AddToWishlistButton/AddToWishlistButton.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from 'react';
+import Button from '@mui/material/Button';
+import axiosInstance from '../../../apis/axiosClient'; // axios instance
+import { AuthStatus, useAuth } from '@akosasante/react-auth-context';
+import { useNavigate } from 'react-router-dom';
+
+const apiBaseURL = process.env.REACT_APP_API_BASE_PATH;
+const wishlistEndpoint = `${apiBaseURL}/wishlist`;
```

Comment:
The axios base instance already takes care of setting up the `REACT_APP_API_BASE_PATH` part (locahost:8000/api/v1 or musicstore.onrender.com/api/v1), so all we need to add here in this file is just the "wishlist" part:

```suggestion
const apiBaseURL = process.env.REACT_APP_API_BASE_PATH;
const wishlistEndpoint = '/wishlist';
```

Reply:
And after that I will need to make changes to my API request?  Here:
`/`${wishlistEndpoint}/${wishlistId}/add_album/${album._id}`/`

Reply:
Or I should just remove this line:
```
- const apiBaseURL = process.env.REACT_APP_API_BASE_PATH;
+ const wishlistEndpoint = '/wishlist';
```

Reply:
Nope; it should still work without needing to change those. Because before; you were setting `wishlistEndpoint` to be `process.env.REACT_APP_API_BASE_PATH` which equaled: `http://localhost:8000/api/v1` on local. Because the address started with the scheme (http) it _**overrides**_ the one that was set in the base axios instance (even though in this case, it's overriding it with basically the same base + `/wishlist`. 

But if we just set `wishlistEndpoint` to only be `/wishlist`, then axios will see that that is not a full URL host; so it will check the instance being used to see if we've set a base URL there (we have [here](https://github.com/Code-the-Dream-School/dd-prac-team4-front/blob/main/src/apis/axiosClient.js#L5)) and it will instead just append `/wishlist` to that. So the end result is the same, but with a smidge less lines of code to be repeated here in this file.

Reply:
> Or I should just remove this line:
> 
> ```
> - const apiBaseURL = process.env.REACT_APP_API_BASE_PATH;
> + const wishlistEndpoint = '/wishlist';
> ```

Yes we can remove `const apiBaseURL = process.env.REACT_APP_API_BASE_PATH;`
And we need just `const wishlistEndpoint = '/wishlist';`

Reply:
Done!